tests/python/gaia-ui-tests/gaiatest/apps/system/regions/status_bar.py

```diff
@@ -7,11 +7,17 @@
 
 
 class StatusBar(Base):
+    _status_bar_locator = (By.ID, 'statusbar')
     _status_bar_time_locator = (By.ID, 'statusbar-time')
     _status_bar_maximized_wrapper = (By.ID, 'statusbar-maximized-wrapper')
     _status_bar_minimized_wrapper = (By.ID, 'statusbar-minimized-wrapper')
 
     @property
+    def is_status_bar_visible(self):
```

Comment:
I don't think it's necessary to precise `status_bar` as we're already using a status bar object